api/resources/translations/messages-es.properties

```
@@ -276,7 +276,7 @@ Weeks\ Pregnant = Semanas de embarazo
 _id = Registro UUID
 about = Acerca
 action.clinic.add = Agregar zona
-action.district_hospital.add = Distrito nuevo
+action.district_hospital.add = Agregar distrito del hospital
```

```suggestion
action.district_hospital.add = Agregar distrito
```
This gives more flexibility in case it changes from `hospital` to something else. Also, it fits better on small screens.